src/app/history/history-bet/history-bet.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="history-container">
```

Comment:
note for angular - no need to use wrapper DIV component for all HTML, read about `:host`

Reply:
done